src/types.rs

```diff
@@ -43,6 +43,7 @@ pub(crate) struct CreateRequest {
 #[derive(Serialize)]
 pub(crate) struct CreateResponse {
 	pub(crate) create_data: CreateData,
+	pub(crate) username: Vec<u8>,
```

Comment:
nit: let's maintain the same order as `ExecuteResponse`
```suggestion
	pub(crate) username: Vec<u8>,
	pub(crate) create_data: CreateData,
```

Reply:
fixed